workflow-service-sdk/pom.xml

```diff
@@ -0,0 +1,203 @@
+        <groupId>dev.parodos</groupId>
+        <artifactId>parados-parent</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>workflow-service-sdk</artifactId>
+    <name>workflow-service-sdk</name>
+    <description>Workflow Service SDK for Parodos</description>
+    <url>https://github.com/redhat-developer/parodos</url>
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <springframework.boot.version>2.5.5</springframework.boot.version>
+        <openapi.generator.version>5.4.0</openapi.generator.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.spring.javaformat</groupId>
+                <artifactId>spring-javaformat-maven-plugin</artifactId>
+                <version>0.0.34</version>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>2.5</version>
+                <executions>
+                    <execution>
+                        <id>auto-clean</id>
+                        <phase>clean</phase>
+                        <goals>
+                            <goal>clean</goal>
+                        </goals>
+                        <configuration>
+                            <filesets>
+                                <fileset>
+                                    <directory>${basedir}</directory>
+                                    <includes>
+                                        <include>**</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>.openapi-generator-ignore</exclude>
+                                        <exclude>pom.xml</exclude>
+                                        <exclude>README.md</exclude>
+                                    </excludes>
+                                    <followSymlinks>false</followSymlinks>
+                                </fileset>
+                            </filesets>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.6.8</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>http://localhost:8080/v3/api-docs</url>
```

Comment:
@pkliczewski @eloycoto 
Any suggestion on how can I get this working?

I'm able to download the JSON file in my local env but it's not working here.

Reply:
what access this port? I'm not sure If I understand the flow.

Reply:
IIUC, for this to work, there is a need to have the workflow-service running.
In https://github.com/springdoc/springdoc-openapi-maven-plugin the expectation is to run this during the integration-test to make sure the service is up and accessible.
Perhaps there is an option to run a goal to start the application for the `/v3/api-docs` to be available, generate the resource and then stop the app.

Reply:
Is there any other way we can get the schema? Running the service may not be that easy.

Reply:
can we commit the openapi schema to a file, and keep it sync in git?

Reply:
yes the server needs to boot and then it read all the annotations on the classes comprising the api , and then generate the schema. It makes sense to generate this and keep it tied to a version. 
So the ci would have to test and make sure the resulting openapi.json doesn't have changes. Developer must make sure the generate ran (don't remeber which mvn target this hooks into. could be a make target to make it simpe)

Reply:
+1 here, we can use something like we had in flotta docs:
https://github.com/project-flotta/project-flotta.github.io/blob/308f37345174dde14be987c901590ff030f3968d/.github/workflows/crd_update.yaml#L24